src/config/pubsub.rs

```diff
@@ -48,6 +61,19 @@ impl Pubsub {
         self.publisher_concurrency
     }
 
+    pub fn read_buffer_size(&self) -> usize {
```

Comment:
Not a blocker, but I wonder if the definitions and these functions should be moved into their own structure which can be shared between `Client` and `Pubsub`.

Reply:
That's a good idea. Will address in a follow-up PR.